src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -33,4 +43,27 @@ public static boolean isNetworkAvailable(final Context context) {
         return connectivityManager.getActiveNetworkInfo() != null && 
             connectivityManager.getActiveNetworkInfo().isConnectedOrConnecting();
     }
+
+    public static void sendFeedEvent(Context context, String category) {
+        Log.d(TAG, "Logging feed event. Category is " + category);
+
+        getTracker(context).send(new HitBuilders.EventBuilder()
+                .setCategory(category)
+                .setAction("click")
+                .build());
+    }
+
+    private static Tracker getTracker(Context context) {
+        if (tracker == null) {
```

Comment:
@myleshorton Yes, but both trackers are logging different events entirely. With the tracker in the app itself, the only event we're currently recording is when a user clicks on an article (nothing related to new sessions or anything).